checker/jdk/index/src/java/io/CharArrayWriter.java

```diff
@@ -201,7 +201,7 @@ public CharArrayWriter append(CharSequence csq) {
      *
      * @since  1.5
      */
-    public CharArrayWriter append(CharSequence csq, @IndexOrHigh("#1") int start, @IndexOrHigh("#1") int end) {
+    public CharArrayWriter append(CharSequence csq, @NonNegative /*!IndexOrHigh("#1")*/ int start, @NonNegative /*!IndexOrHigh("#1")*/ int end) {
```

Comment:
What is the purpose of all these `/*!...*/` comments?

Reply:
The purpose is to retain the information about the upper bounds of the annotated parameters in the source code, without the upper bounds being actually checked. The `/*!` comments are already used for this purpose elsewhere.

Reply:
How about adding a `checker/jdk/index/README` file that documents this syntax?